apps/files_sharing/src/views/SharingDetailsTab.vue

```diff
@@ -278,7 +278,7 @@ export default {
 
 	computed: {
 		title() {
-			let title = t('files_sharing', 'Share with ')
+			let title = t('files_sharing', 'Share with')
 			if (this.share.type === this.SHARE_TYPES.SHARE_TYPE_USER) {
 				title = title + this.share.shareWithDisplayName
```

Comment:
correct me if i am wrong, but the space is missing then here where it is concatenated with the display name.  At least here, didn't check further.
```suggestion
			let title = t('files_sharing', 'Share with') + " "
			if (this.share.type === this.SHARE_TYPES.SHARE_TYPE_USER) {
				title = title + this.share.shareWithDisplayName
```